generate.py

```diff
@@ -25,4 +30,13 @@ def generate_custodian_policy(self):
 
     # Generates the terraform file that will deploy the IAM roles and policies needed by custodian
     def generate_terraform(self):
-        return iam_template.terraform_iam_template(self.config)
+        generated_terraform = iam_template.terraform_iam_template(self.config)
+        filepath = os.path.join(ConfigGenerator.TERRAFORM_GENERATED_DIR, ConfigGenerator.TERRAFORM_FILE)
```

Comment:
Nit: Shorter and potentially clearer to use: `filepath = os.path.join(self.TERRAFORM_GENERATED_DIR, self.TERRAFORM_FILE)`

Reply:
Switched. 

Not disagreeing, but wouldn't accessing class variables via the class rather than instance be more 'standard'? I know setting is a different story, just curious.